entity-framework/efcore-and-ef6/porting/port-detailed-cases.md

```diff
@@ -66,7 +66,7 @@ EF6 supported table-per-hierarchy (TPH), table-per-type (TPT) and table-per-conc
 
 ### Attributes
```

Comment:
"For spatial support, EF Core recommends using a third-party library such as NetTopologySuite." I would change this to indicate that EF Core integrates with NTS. Using a different third-party library won't work unless this same integration work is done, which is non-trivial. Something like, "For spatial support, EF Core integrates with the third-party community library NetTopologySuite."